### PR TITLE
Comments added to single page article

### PR DESCRIPTION
**Pull request by milbot1992**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -10,12 +11,15 @@ function App() {
...
       <Nav />
-      <Routes>
-        <Route path="/" element={<ArticleList topic="all" />} />
-        <Route path="/coding" element={<ArticleList topic="coding" />} />
-        <Route path="/football" element={<ArticleList topic="football" />} />
-        <Route path="/cooking" element={<ArticleList topic="cooking" />} />
-      </Routes>
+      <div className = 'content-container'>
+        <Routes>
+          <Route path="/news" element={<ArticleList topic="all" />} />
```
Comment (B7Bprog) [inline src/App.jsx:16]: Do you need all theses separate routes for each topic? What if you updated the ArticleList component when a topic is selected? Could you use a parametric endpoint here?

File: src/components/SingleArticle.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import { useEffect, useState, Fragment } from "react";
+import { getArticleById } from "../api";
+import { getCommentsById } from "../api";
+import CommentCard from "./CommentCard";
+import formatDistanceToNow from 'date-fns/formatDistanceToNow';
+
+export default function SingleArticle() {
+    const { article_id } = useParams();
+    const [article, setArticle] = useState({});
+    const [comments, setComments] = useState([])
```
Comment (B7Bprog) [inline src/components/SingleArticle.jsx:11]: Please add a state for loading and display a message to the user while the single article is loading, and one for the comments as well.

File: src/components/SingleArticle.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            id="medium-image"
+        />
+        <p className = 'single-article-body'>{body}</p>
+        <div className = 'single-article-votes'>
+            <h3>Votes: {votes} </h3>
+            <h3>Comments: {comment_count}</h3>
+            <h3>{topic}</h3>
+        </div>
+
+        <ul className = 'comments-list'>
```
Comment (B7Bprog) [inline src/components/SingleArticle.jsx:58]: What would you display to the user where there are no comments for an article yet?

**Review by B7Bprog — COMMENTED** [review]
Hey Millie, only a few things to refactor, please see my comments.

File: src/App.jsx (hunk 1 above)
Comment (milbot1992) [reply src/App.jsx:16]: Yes much nicer, thank you :)

**Pull request merged**